Composer/packages/client/src/pages/language-generation/index.tsx

```diff
@@ -35,7 +35,12 @@ const LGPage: React.FC<RouteComponentProps> = props => {
   const matched = /^\/language-generation\/(\w+)/.exec(path.replace(/edit(\/)*$/, ''));
   const fileId = matched ? matched[1] : '';
   const edit = /edit(\/)*$/.test(path);
-  const lgFile = lgFiles.length ? lgFiles[0] : null;
+  const file = lgFiles?.find(({ id }) => id === 'common');
+
+  useEffect(() => {
+    // fileId not found, redirect to common
+    if (!fileId) navigateTo('/language-generation/common');
```

Comment:
Couple of things:

1. Could this be done with the router instead?
2. If not, we should use something like `replaceState` instead so this uri is not in the history stack.